charts/wasp-open-api-v2/templates/cronjob.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: {{ include "common.capabilities.cronjob.apiVersion" . }}
```

Comment:
Is there a particular bitnami chart you've taken the cronjob from? Just looking to make sure we continue following best practices.

Reply:
The etcd chart has a cronjob.

Reply:
Is this copied from that then?

Reply:
Heavily inspired by.